egs2/TEMPLATE/spk1/spk.sh

```diff
@@ -262,8 +283,17 @@ if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
             utils/copy_data_dir.sh --validate_opts --non-print data/"${train_set}" "${data_feats}/${train_set}"
 
             # copy extra files that are not covered by copy_data_dir.sh
-            # category2utt will be used bydata sampler
+            # category2utt will be used by the data sampler
             cp data/"${train_set}/spk2utt" "${data_feats}/${train_set}/category2utt"
+
+            # copy files for spoofing task if multi-task sasv is enabled
+            if "${multi_task}" && "${sasv_task}"; then
+                cp data/"${train_set}/utt2spf" "${data_feats}/${train_set}/utt2spf"
+                cp data/"${train_set}/spf2utt" "${data_feats}/${train_set}/spf2utt"
+                # category2utt_2 will also be used by the data sampler
+                cp data/"${train_set}/spf2utt" "${data_feats}/${train_set}/category2utt_2"
```

Comment:
what is `category2utt_2` for? Can you update the name to reflect what it contains?

Reply:
Since category2utt was generic in naming, I assumed the other one could also be generic.
Since it's only seen by the sampler, which can be generically used.
But I can update the name. Should I update the name to spf2utt?